.palantir/revapi.yml

```diff
@@ -1136,6 +1136,78 @@ acceptedBreaks:
       new: "method org.apache.iceberg.BaseMetastoreOperations.CommitStatus org.apache.iceberg.BaseMetastoreTableOperations::checkCommitStatus(java.lang.String,\
         \ org.apache.iceberg.TableMetadata)"
       justification: "Removing deprecated code"
+    - code: "java.method.returnTypeTypeParametersChanged"
```

Comment:
Unfortunately I think we have to keep the old API's for one release.  Let's just make a new method then.  Hopefully we can re-use logic between these.

Reply:
@szehon-ho Sounds good, let me make a new method.

Reply:
@szehon-ho I added a new `snapshotManifestPairs()` to the base classes.